bin/network-monitor/src/monitor/tasks.rs

```diff
@@ -214,7 +219,45 @@ impl Tasks {
         faucet_rx
     }
 
+    /// Spawn the counter increment task.
+    #[instrument(target = COMPONENT, name = "tasks.spawn-counter", skip_all)]
+    pub fn spawn_counter_increment(&mut self, config: &MonitorConfig) -> Receiver<ServiceStatus> {
```

Comment:
nit: Since the service name is "Network Transactions", maybe this could be named similarly


```suggestion
    pub fn spawn_network_transactions_service(&mut self, config: &MonitorConfig) -> Receiver<ServiceStatus> {
```